providers/frame/zombienet.ts

```diff
@@ -58,6 +63,20 @@ export class ZombienetProvider extends FrameProxyProvider {
   zombienet(configPath: string): Promise<Network> {
     return this.zombienetMemo.run(configPath, async () => {
       const tmpDir = Deno.makeTempDirSync({ prefix: `capi_zombienet_` })
+      const watcher = Deno.watchFs([tmpDir])
+      const networkManifestPath = path.join(tmpDir, "zombie.json")
+      const configPending = deadline(
+        (async () => {
+          for await (const e of watcher) {
+            if (e.kind === "create" && e.paths.some((path) => path.endsWith(networkManifestPath))) {
```

Comment:
Why `.endsWith`?

Reply:
For some reason, these paths are prefixed with `/private/`, whereas the `networkManifestPath` is not. Will test to make sure this is true in other envs.

Reply:
The specified watch dir is `/var/...`, yet the events' `paths` get populated with `/private/var/...`. I believe this is related to denoland/deno#10678, although there's no mention of this prefixing behavior elsewhere. `.endsWith` should be a reasonable safeguard, incase "private" is (for some reason) another name on different systems (or not present at all).

Reply:
`/var/` is likely symlinked to `/private/var/` on your system (as it is on mine)
```ts
const networkManifestPath = await Deno.realPath(path.join(tmpDir, "zombie.json"))
```